components/suggestion/suggestion_box.jsx

```diff
@@ -196,12 +202,10 @@ export default class SuggestionBox extends React.Component {
             // iOS doesn't support e.relatedTarget, so we need to use the old method of just delaying the
             // blur so that click handlers on the list items still register
             if (this.presentationType !== 'date' || this.props.value.length === 0) {
-                setTimeout(() => {
-                    GlobalActions.emitClearSuggestions(this.suggestionId);
-                }, 200);
+                this.handleEmitClearSuggestions(this.suggestionId);
```

Comment:
This case still needs a 200 ms timeout to stop the list from being cleared before the click action has been handled